app/_src/guides/consumer-producer-policies.md

```diff
@@ -0,0 +1,231 @@
+Producer policies will be created in the same zone as MeshService they target. Producer policies will be synced to other zones. 
+
+**Consumer** is someone that utilizes a service by communicating with it. Consumer policies will be applied in consumer 
+namespace and will target MeshService from different namespace. Consumer policy will take effect only in consumer namespace.
+Consumer policies will override producer policies.
+
+## Testing producer policy
+
+To test MeshTimeout that we've applied in previous steps we need to simulate delays on our requests. To do this we need
+add header `x-set-response-delay-ms` to our requests
+
+We can now make few requests to our demo-app, and we should see timeouts:
+
+```shell
+kubectl exec -n first-consumer consumer -- curl -s -XPOST demo-app.kuma-demo:5000/increment -H "x-set-response-delay-ms: 2000"
+```
+
+Example output:
+
+```
+{"counter":"3","zone":"local","err":null}
+```
+
+We should see the same results when making requests from second-consumer namespace:
+
+```shell
+kubectl exec -n second-consumer consumer -- curl -s -XPOST demo-app.kuma-demo:5000/increment -H "x-set-response-delay-ms: 2000"
+```
+
+## Utilizing consumer policy
+
+To this moment we were relying on producer policy created previously. Let's assume that we don't mind waiting a little longer
+for the response from demo-app, and we would like to override the timeout just for our namespace. To do so we need to create new policy:
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: consumer-timeout
+  namespace: first-consumer
+  labels:
+    kuma.io/mesh: default
+    kuma.io/origin: zone
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: demo-app
+        namespace: kuma-demo
+      default:
+        http:
+          requestTimeout: 3s" | kubectl apply -f -
+```
+
+When we now make requests from the first-consumer namespace all the requests should succeed, but they will take longer:
+
+```shell
+kubectl exec -n first-consumer consumer -- curl -s -XPOST demo-app.kuma-demo:5000/increment -H "x-set-response-delay-ms: 2000"
+```
+
+Policy that we have just applied is consumer policy. This means that traffic from other namespaces to demo-app should not be affected by it.
+We can test this by making requests from our second-consumer namespace:
+
+```shell
+kubectl exec -n second-consumer consumer -- curl -s -XPOST demo-app.kuma-demo:5000/increment -H "x-set-response-delay-ms: 2000"
+```
+
+We should still see timeouts:
+
+```
+upstream request timeout
+```
+
+## What we've learnt?
+
+- How to apply policy on a namespace
+- What are policy roles and what types can you encounter 
+- What are producer and consumer policies
+- How producer and consumer policies interact with each other
+
+## Next steps
+
+- Read more about [producer/consumer policies](TODO)
```

Comment:
It will be in:

```suggestion
- Read more about [producer/consumer policies](/docs/{{ page.version }}/policies/introduction)
```

Reply:
Maybe it's worth cross linking to another guide like: `federate` no?

Reply:
linked federation guide